packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -235,6 +263,13 @@ export class SOQLEditorInstance {
   protected async retrieveSObject(sobjectName: string): Promise<void> {
     return withSFConnection(async conn => {
       conn.describe$(sobjectName, (err, sobject) => {
+        if (err) {
+          const message = nls.localize(
+            'error_sobject_metadata_request',
+            sobjectName
+          );
+          channelService.appendLine(message);
+        }
```

Comment:
After Fernando's Pr merge, i'll align this with his changes.  And include his check for `if (e.errorCode !== 'NOT_FOUND')`.  Otherwise, message will look like:

<img width="1212" alt="sobject-metadata-request" src="https://user-images.githubusercontent.com/599418/101826811-bd722780-3aec-11eb-99da-907af1a45d75.png">